src/card.tsx

```diff
@@ -516,6 +518,13 @@ export const MediaCard = React.forwardRef<HTMLDivElement, MediaCardProps>(
                                     buttonLink={buttonLink}
                                 />
                             </div>
+                            {asset ? (
+                                <Box className={styles.mediaCardAsset} paddingX={16} paddingY={16}>
+                                    {asset}
+                                </Box>
+                            ) : (
+                                <Box paddingBottom={actions?.length || onClose ? 64 : 0} />
```

Comment:
This is not needed. It's needed in other cards like DisplayMediaCard because the `asset` is not absolutely positioned like here.